weldx/transformations/local_cs.py

```diff
@@ -303,6 +306,29 @@ def _build_coordinates(coordinates, time: pd.DatetimeIndex = None):
         xarray.DataArray
 
         """
+        if isinstance(coordinates, TimeSeries):
+            if coordinates.is_expression:
+                return coordinates
+            if coordinates.shape[1] != 3:
+                raise ValueError(
+                    "The shape of the TimeSeries must be (n, 3). It actually is: "
+                    f"{coordinates.shape}"
+                )
+            coordinates = coordinates.data_array
+            # This is a workaround to remove the warning about stripped units. This line
+            # should be removed once we add/require units in the lcs
+            # Additionally, the correct unit should be checked for TimeSeries
+            # (also expressions)
+            coordinates.data = coordinates.data.to(UREG.mm).m
+            if coordinates.data.shape[0] == 1:
+                coordinates = coordinates.data.reshape(3)
+            else:
+                coordinates = coordinates.rename({coordinates.dims[1]: "c"})
+                coordinates = coordinates.assign_coords(dict(c=["x", "y", "z"]))
```

Comment:
I will put it into a static method. Also, I would deprecate `utils.lcs_coords_from_ts`, because all its functionality (except for the interpolation) is now handled by the LCS